sql/core/src/test/resources/sql-tests/inputs/subquery/any-subquery/any-multiple-columns.sql

```diff
@@ -0,0 +1,66 @@
+-- A test suite for ANY/SOME predicate with multiple columns
+
+create temporary view a as select * from values
+  (1, 1),
+  (null, 2),
+  (3, 3),
+  (4, null)
+  AS a(a1, a2);
+
+create temporary view b as select * from values
+  (1, 1),
+  (4, null),
+  (3, 2)
+  AS b(b1, b2);
+
+create temporary view c as select * from values
+  (1, 1),
+  (null, null),
+  (3, 2)
+  AS c(c1, c2);
+
+-- ANY predicate with `multi-column = subquery`
+-- TC 01.01
+SELECT a1,
+       a2
+FROM   a
+WHERE  (a1, a2) = ANY (SELECT b1,
+                              b2
+                       FROM   b );
```

Comment:
plz remove unnecessary spaces, e.g.,`b )` -> `b)`. Also, could you fix all the other same cases?

Reply:
ok